docs/management/admin/response-actions-config.asciidoc

```diff
@@ -25,66 +24,107 @@ preview::[]
 
 * <<endpoint-management-req,{elastic-sec} feature privileges>>: **All** for the response action features, such as **Host Isolation**, that you want to perform.
 
-* Endpoints must have actively running SentinelOne agents installed.
+* Endpoints must have actively running third-party agents installed.
 --
 
-Configuration requires the following general steps. Expand the steps and follow the links for detailed instructions:
+Expand a section below for your endpoint security system:
 
-. **Generate API access tokens in SentinelOne.** You'll need these tokens in later steps, and they allow {elastic-sec} to collect data and perform actions in SentinelOne. 
+.**Set up CrowdStrike response actions**
+[%collapsible]
+====
+// NOTE TO CONTRIBUTORS: These sections have very similar content. If you change anything 
+// in this section, apply the change to the other sections, too.
+
+. **Create an API client in CrowdStrike.** Refer to CrowdStrike's docs for instructions on creating an API client.
++
+- Give the API client the minimum privilege required to read CrowdStrike data and perform actions on enrolled hosts.
+- Take note of the client ID, client secret, and base URL; you'll need them in later steps when you configure {elastic-sec} components to access CrowdStrike.
+
+. **Install the CrowdStrike integration and {agent}.** Elastic's {integrations-docs}/crowdstrike[CrowdStrike integration]
```

Comment:
NOTE: there are two crowdstrike integrations. Lets confirm that the only needed one is this one you documented here (the other one is called 'CrowdStrike Falcon Intelligence')